pipelines/ApPipe.yaml

```diff
@@ -65,8 +65,8 @@ contracts:
       transformDiaSrcCat.connections.ConnectionsClass(config=transformDiaSrcCat).diaSourceCat.name
   - imageDifference.connections.ConnectionsClass(config=imageDifference).subtractedExposure.name ==
         diaPipe.connections.ConnectionsClass(config=diaPipe).diffIm.name
-  - imageDifference.connections.ConnectionsClass(config=imageDifference).warpedExposure.name ==
-        diaPipe.connections.ConnectionsClass(config=diaPipe).warpedExposure.name
+  - imageDifference.connections.ConnectionsClass(config=imageDifference).template.name ==
```

Comment:
When I was running with this ticket branch for DM-33911 I was getting an error on BPS submission due to this line. If I instead did

imageDifference.connections.ConnectionsClass(config=imageDifference).inputTemplate.name ==
         diaPipe.connections.ConnectionsClass(config=diaPipe).template.name
it works.